Matrix_multiplication/matrix.temper.md

```diff
@@ -15,6 +17,47 @@ And actually check the string value because that's easier for now.
       assert(m == "-7.0, -6.0, 11.0\n-17.0, -20.0, 25.0");
     }
 
+## Benchmark
+
+Also provide a function for benchmarking matrix multiplication. Just keep the
+running of it disabled by default.
+
+    // benchmark(1e6.toIntUnsafe());
```

Comment:
On my machine, node actually crashes at 1e7 here. But for most backends, the per-row version is just a lot slower than the dense array version.